ledger/acctonline_test.go

```diff
@@ -375,10 +375,10 @@ func TestAcctOnlineCache(t *testing.T) {
 	_, totals, err := au.LatestTotals()
 	require.NoError(t, err)
 
+	// check cache was initialized with db state
 	for _, bal := range allAccts {
-		// force populate the cache
-		oad, err := oa.lookupOnlineAccountData(0, bal.Addr)
```

Comment:
keep the `oa.lookupOnlineAccountData` call to check it also returns the same value

Reply:
We don't need this since it is exactly the same as the previous test.